e2e/rbd.go

```diff
@@ -432,12 +432,35 @@ var _ = Describe("RBD", func() {
 					if err != nil {
 						e2elog.Failf("failed to load PVC with error %v", err)
 					}
-
+					label := make(map[string]string)
 					pvc.Namespace = f.UniqueName
 					err = createPVCAndvalidatePV(f.ClientSet, pvc, deployTimeout)
 					if err != nil {
 						e2elog.Failf("failed to create PVC with error %v", err)
 					}
+					app, err := loadApp(appPath)
+					if err != nil {
+						e2elog.Failf("failed to load app with error %v", err)
+					}
+					// write data in PVC
+					label[appKey] = appLabel
+					app.Namespace = f.UniqueName
+					app.Spec.Volumes[0].PersistentVolumeClaim.ClaimName = pvc.Name
+					wErr := writeDataInPod(app, f)
+					if wErr != nil {
+						e2elog.Failf("failed to write data in the pod with error %v", err)
+					}
```

Comment:
re-use the err which is already present.

```suggestion
err = writeDataInPod(app, f)
if err != nil {
e2elog.Failf("failed to write data in the pod with error %v", err)
}
```